src/optics/lib/Convex/CardanoApi/Lenses.hs

```diff
@@ -208,6 +218,24 @@ instance IsBabbageEraOnwards C.BabbageEra where
 instance IsBabbageEraOnwards C.ConwayEra where
    babbageEraOnwards = C.BabbageEraOnwardsConway
 
+txBody :: Getter (C.Tx era) (C.TxBody era)
+txBody = L.to get_ where
+  get_ (C.Tx txb _) = txb
+
+txBodyContent :: Getter (C.TxBody era) (C.TxBodyContent C.ViewTx era)
+txBodyContent = L.to get_ where
+  get_ (C.TxBody txbc) = txbc
```

Comment:
Can this be turned into a `Lens`, maybe with some extra constraints on `era`?

Reply:
Actually, `C.TxBody` and `C.TxBodyContent` are more or less isomorphic types (going from `TxBodyContent` to `TxBody` returns an `Either`).

Can we use `Iso` in this case?

Reply:
No, if it returns `Either` then it should be a `Prism`. `Iso a b` only if we have `f :: a -> b` and `g :: b -> a` such that `f . g = id` and vice versa